### PR TITLE
Execute browser tests for Edge #671

### PR DESCRIPTION
**Pull request by HeikoKlare**
Browser tests were only executed for the default configuration of a system's browser using the `SWT.NONE` flag. Other configurations, such as using the Edge browser in Windows, were not tested.

This change parameterizes the browser tests to also execute them for the Edge browser on Windows. It also deactivates those tests for the Edge browser for which the implementation does (currently) not work. This allows to detect regressions when performing future changes to the Edge browser.

Fixes https://github.com/eclipse-platform/eclipse.platform.swt/issues/671

### CODE REVIEW

File: tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java [hunk 1]
```diff
@@ -206,6 +229,16 @@ public void tearDown() {
...
+	if (isEdge) {
+		// wait for and process pending events to properly cleanup Edge browser resources
+		do {
+			processUiEvents();
+			try {
+				Thread.sleep(100);
+			} catch (InterruptedException e) {
+			}
+		} while (Display.getCurrent().readAndDispatch());
+	}
```
Comment (HeikoKlare) [inline tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java:241]: This is necessary because otherwise some OS events that are somehow produced by one test occur and are processed by the next test and may interfere with the Edge browser instantation. I have invested quite some time to identify where the events come from and how to properly handle them, but I did not find a proper way to do so. If someone is able to identify the reasons for the events and how to properly process them, I would be glad to have a better solution than this,

**Comment by vogella** [comment]
@HeikoKlare can this be merged?

**Comment by jukzi** [comment]
```
creating browser took too long: 23469ms
java.lang.AssertionError: creating browser took too long: 23469ms
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.eclipse.swt.tests.junit.Test_org_eclipse_swt_browser_Browser.createBrowser(Test_org_eclipse_swt_browser_Browser.java:290)
	at org.eclipse.swt.tests.junit.Test_org_eclipse_swt_browser_Browser.setUp(Test_org_eclipse_swt_browser_Browser.java:173)

```

**Comment by HeikoKlare** [comment]
I converted this back to draft due to the recent test failure and I will have another look at it.
Further increasing the waiting time between test cases would probably "solve"(/hide) the problem, but I'd prefer a solution that makes waiting times between test cases obsolete.

**Comment by fedejeanne** [comment]
Test failures in Linux are unrelated #1564
